ONSdigital/eq-author-app
# Added section intro title & content validation

### What is the context of this PR?
Validation needs to be added to section page ensure page is invalid if introduction title is entered without an introduction section and vice versa.

### How to review 
1. Create section page
2. Add introduction title - introduction content should be highlighted as invalid
3. Add introduction content - section should be valid
4. Remove introduction title - introduction title should be highlighted as invalid
5. Remove introduction content - section should be valid

Observe the error count on navigation changing appropriately for each step.
